aminator/plugins/provisioner/apt.py

```diff
@@ -47,23 +47,15 @@ class AptProvisionerPlugin(BaseProvisionerPlugin):
     """
     _name = 'apt'
 
-    def _refresh_repo_metadata(self):
```

Comment:
This wasn't an abstract method and the layer of indirection didn't seem to be helping anything so I have things just call apt_get_update()